public/app/features/explore/Logs/LogsMetaRow.tsx

```diff
@@ -88,9 +88,39 @@ export const LogsMetaRow = memo(
           break;
         case DownloadFormat.CSV:
           const dataFrameMap = new Map<string, DataFrame>();
+
+          let sortedValue = new Map<string, any | undefined[]>();
```

Comment:
@yincongcyincong I'm having a hardtime understanding your solution here with this map.